package.json

```diff
@@ -14,38 +14,35 @@
     "watch": "npm run watch:css",
     "build-dev": "npm run build-dev:css",
     "watch-dev": "npm run watch-dev:css",
-
     "build:css": "npm run build:less",
     "watch:css": "npm run watch:less",
     "build-dev:css": "npm run build-dev:less",
     "watch-dev:css": "npm run watch-dev:less",
-
     "build:less": "grunt less && npm run lessToSass",
     "watch:less": "grunt watch:less",
     "build-dev:less": "grunt lessdev && npm run lessToSass",
     "watch-dev:less": "grunt watch:lessdev",
-
     "build:scss": "grunt scss",
     "watch:scss": "grunt watch:scss",
-
     "lessToSass": "grunt lessToSass",
     "watch:lessToSass": "grunt watch:lessToSass"
   },
   "dependencies": {
+    "@splidejs/splide": "^4.1.4",
     "autocomplete.js": "vufind-org/autocomplete.js#v2.1.5",
     "grunt": "^1.0.4",
     "grunt-contrib-less": "^2.0.0",
     "grunt-less-to-sass": "duvillierA/grunt-less-to-sass#0.0.11",
     "grunt-sass": "^3.1.0",
     "jit-grunt": "^0.10.0",
-    "node-sass": "^7.0.1"
+    "node-sass": "^7.0.1",
+    "vanilla-cookieconsent": "3.0.0-rc.14"
```

Comment:
Why did this move up from devDependencies to dependencies? I think the idea is that dependencies that get installed by Phing are treated as devDependencies, because we only need them when developing (since the files are committed to Git for the benefit of "regular users"). For this reason, we should move splide down there as well.

Reply:
dependencies are for libraries used on the front-end, devDependencies are for build tools. It's me being a bit strict and it's not the best time to do this, and there's clearly a lot more here to cleanup.

Reply:
I think it's a bit murky because of the way VuFind uses Javascript. Technically, it could be argued that EVERYTHING is a build tool. You don't need to run `npm install` to run the VuFind application, because everything is already there in Git. You ONLY need Javascript tools if you're doing development of some sort. And by the standard you propose, some things currently in `dependencies` are dev tools (grunt, node-sass, etc.) and some of the things in devDependencies are not (jstree). I think we should talk this through and define a standard that we can all agree on -- probably with input from @EreMaijala when he's back from holiday -- but for now I'd suggest just adding splide wherever you think it belongs, but avoiding moving other stuff around until we can address the moves more holistically.

Reply:
I'd draw the line between non-development needs and development needs this way (and I totally understand this being arbitrary and non-scientific):
- Non-development tools are needed to customize a VuFind installation including branding, colors etc. but not for actual development work.
- Development dependencies are needed to update js libraries etc. that could also require code changes to accommodate upstream changes.

As far as I can see, with this distinction our current package.json in dev makes sense for most parts, but autocomplete.js is in wrong block since it's only needed for updating of the library.

Reply:
Thanks, @EreMaijala, I think that rule of thumb seems workable. If @crhallberg agrees, maybe he can move the autocomplete down instead of the cookie-consent up, and then we'll be back to consistency. :-)

Reply:
I'd argue that autocomplete is in the right spot and cookieconsent needs to be moved up because they are needed to run VuFind successfully. Anything moved by copynodemodules should be in non-development tools.

Reply:
But I'd say that copynodemodules is a development tool (upgrading dependencies is not a simple task typically done for customizing VuFind), and we don't need these otherwise.

Reply:
Maybe the best thing for the present would be to go with @EreMaijala's approach (since that's what I've been telling people to do, and that's how the majority of the dependencies are currently organized) and we can revisit this as part of the JS tooling overhaul, which is on the roadmap for 10.0.

Reply:
I believe that #3290 has introduced a much better solution to the package.json problem. @crhallberg, can you update this PR to move the splide-related installation logic to the places introduced there?